holidayextras/ui-toolkit
# Adding to our Eslint

#### What does this PR do? (please provide any background)
We don't really have a JS style guide for the UI-Toolkit so we have added extras linting to ensure code consistency.

#### What tests does this PR have?
None

#### How can be this tested?
- We are using `1tbs` js brace styles, there are example on http://eslint.org/docs/rules/brace-style.html on what will pass and fail linting.
- Write or adjust some JS in the toolkit locally then `npm run lint` and you should see lint pass or fail depending on the code you have written.

#### Screenshots / Screencast
![screen shot 2015-05-29 at 16 41 29](https://cloud.githubusercontent.com/assets/778942/7886377/96557ffe-0621-11e5-98aa-207ef7e19144.png)

#### What gif best describes how you feel about this work?
![friday](http://media.giphy.com/media/yoAxHFxP1npEQ/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the contributing document and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
